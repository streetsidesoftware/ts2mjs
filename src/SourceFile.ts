export interface SourceFile {
    srcFilename: string;
    content: string;
    map?: SourceMap | undefined;
}

export interface SourceMap {
    url: URL;
    map: string;
}
