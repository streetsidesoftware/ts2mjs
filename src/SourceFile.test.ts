import { describe, expect, test } from 'vitest';

import * as SourceFile from './SourceFile.js';

describe('SourceFile', () => {
    test('SourceFile Api', () => {
        expect(SourceFile).toMatchSnapshot();
    });
});
