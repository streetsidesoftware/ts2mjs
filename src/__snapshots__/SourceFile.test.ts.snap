// Vitest Snapshot v1

exports[`SourceFile > SourceFile Api 1`] = `{}`;
